src/components/label/label.svelte

```diff
@@ -0,0 +1,67 @@
+<script context="module" lang="ts">
+  export const colors = [
+    'gray',
+    'secondary',
+    'primary',
+    'red',
+    'yellow',
+    'green',
+    'blue',
+    'purple',
+    'pink',
+    'teal',
+    'hero'
```

Comment:
`hero` as a color should actually be `CTA`. `heavy` mode should be called `hero`

Reply:
Renamed to `CTA`